test/presenters/supergroups/guidance_and_regulation_test.rb

```diff
@@ -102,4 +102,12 @@
       assert_equal expected, guidance_and_regulation_supergroup.promoted_content(taxon_id)
     end
   end
+
+  describe "document types" do
+    it "returns appropriate things" do
+      document_types = GovukDocumentTypes.supergroup_document_types("guidance_and_regulation")
+
+      assert document_types.include? "guidance"
```

Comment:
How about we use the `assert_includes` assertion?

```rb
assert_includes document_types, "guidance"
```

At the moment our failure message will be "Expected false to be truthy.", with a more specific assertion our error message would be "Expected 'document_types' to include 'guidance'".

Reply:
Good call, will update, thanks